R-package/DESCRIPTION

```diff
@@ -62,5 +62,5 @@ Imports:
     parallel,
     utils
 SystemRequirements:
-    C++11
+    ~~CXXSTD~~
```

Comment:
The permission of this file also changed: "100755 → 100644", is that expected?

Reply:
thanks for catching that!

I used GitHub's in-browser merge conflict resolution after merging #5341 . Maybe it changed the permissions.

Fixed in https://github.com/microsoft/LightGBM/pull/5690/commits/79ee233ba7a1605722ed41984bef21e35cb95970